tls/extensions/s2n_client_supported_groups.h

```diff
@@ -19,8 +19,12 @@
 #include "tls/extensions/s2n_extension_type.h"
 #include "tls/s2n_connection.h"
 
+#define S2N_SUPPORTED_GROUP_SIZE 2
+
 extern const s2n_extension_type s2n_client_supported_groups_extension;
 bool s2n_extension_should_send_if_ecc_enabled(struct s2n_connection *conn);
 
+S2N_RESULT s2n_client_supported_groups_parse_groups_count(struct s2n_stuffer *extension, uint16_t *count);
+
```

Comment:
The duplication of groups looks a little silly, but I don't have a strong objection to it if you really like it.
```suggestion
S2N_RESULT s2n_client_supported_groups_parse_count(struct s2n_stuffer *extension, uint16_t *count);

```